db/migrations/postgres/000024_create_token_tables.up.sql

```diff
@@ -0,0 +1,12 @@
+BEGIN;
+CREATE TABLE tokenpool (
+  seq            SERIAL          PRIMARY KEY,
+  pool_id        VARCHAR(80)     NOT NULL,
```

Comment:
pool_id should be a FireFly-generated UUID
on-chain ID should be another field (protocol_id?) - see transactions table for one example